src/RNQuery/RNQueryPage.jsx

```diff
@@ -0,0 +1,44 @@
+import {useQuery} from "react-query";
+
+const fetchApi = async() => {
+    return await fetch(`${process.env.REACT_APP_API_URL}`).then((res)=> res.json());
+}
+
+export const RNQueryPage = () => {
+
+    const onSuccess = (data) => {
+        console.log("fetch is done", data);
```

Comment:
we can skip on success and on error if we are just using for log

Reply:
removed.